tests/test_pipeline/test_hierarchical_pipeline.py

```diff
@@ -5,7 +5,10 @@
 import pytest
```

Comment:
May you also add test for `get_level_dataset` and `reconcile` with quantiles different from target

Reply:
These tests should be in folders for dataset and reconciler, not in pipeline